### DISCUSSION TITLE
SD_Dreambooth

### DISCUSSION THREAD

**Discussion by simonmcnair** [AbdBarho/stable-diffusion-webui-docker#467] [Q&A]
Whenever I try and install dreambooth stuff breaks.  Not Just in Docker, but generally.  I got it working in windows, but then the move to Linux I couldn't get it working, and neither can I in docker.

I would love some advice.
The errors I normally get are either pytorch version mismatches with cuda or illegal instruction, and then I have to remove the extension and completely rebuild to container before it will work.

If anyone can offer any advice I'd appreciate it.

**Comment by alickwong** [comment]
same here, i got this error message
```
To create a public link, set `share=True` in `launch()`.
*** Error executing callback app_started_callback for /stable-diffusion-webui/extensions/sd_dreambooth_extension/scripts/api.py
    Traceback (most recent call last):
      File "/stable-diffusion-webui/modules/script_callbacks.py", line 126, in app_started_callback
        c.callback(demo, app)
      File "/stable-diffusion-webui/extensions/sd_dreambooth_extension/scripts/api.py", line 530, in dreambooth_api
        model_cfg: DreamboothConfig = Body(description="The config to save"),
    NameError: name 'DreamboothConfig' is not defined
```
